test/uk/gov/hmrc/gform/models/FieldValueGroupRawSpec.scala

```diff
@@ -26,7 +26,8 @@ class FieldValueGroupRawSpec extends FlatSpec with Matchers with EitherValues wi
   val gfvr = FieldValueRaw(
     id = FieldId("groupid"),
     `type` = Some(GroupRaw),
-    label = "Group Label"
+    label = "Group Label",
+    format = Some(TextFormat("vertical"))
```

Comment:
There is a nicer cats syntax available after importing `import cats.syntax.all._`. Consider:
`"http:\\schemaId".some` vs `Some("http:\\schemaId")`